lib/src/upgrader.dart

```diff
@@ -223,6 +224,12 @@ class Upgrader {
         print('upgrader: countryCode: $code');
       }
 
+      // If this platform is not iOS, skip the iTunes lookup
+      // Get android update without appcast
+      if (Platform.isAndroid) {
+        await _getAndroidStoreVersion();
+      }
+
       final iTunes = ITunesSearchAPI();
```

Comment:
Why does ITunesSearchAPI still get called for Android? There is no need to use ITunesSearchAPI on Android. This logic should have an else if Platform.isIOS.